### ISSUE TITLE
Update to "Alarming Hours" report

### ISSUE THREAD

**Issue by cvoorhees** [magfest/ubersystem#913]
Eli, et. al.

Can we add a section on the "Alarming Hours" report that is basically a massive inner join of the following:
- Staffers
- Signed up for 0 hours
- Assigned to departments that are shifted (ie not marketplace or concerts, tho we need to fix that next year).
- Accepted their badge (ie not placeholders)

That way we can see who says they are coming and hasn't signed up for any shifts. If this could be done relatively quickly that would be awesome so I can send out emails and/or we could configure an automated email. 

If we do an automated email the text could read something like:

Dear <staffer>,

You have accepted your staff badge for MAGFest 13 but you have not signed up for any shift hours. To maintain your staff status you must sign up and work at least 18 weighted hours. Please use the link below to sign up for shifts. 

<link>

Sincerely,
Staffing Operations

---

Thanks!
-Carla

**Comment by EliAndrewC** [comment]
We already have the following automated emails:
- an email for people with Staff badges signing up for hotel rooms who have not yet met their 30 hour requirements (one of these has already gone out and there will be another one closer to MAGFest)
- an email for anyone who checked the box that says they want to volunteer who hasn't signed up for any hours yet

If I understand this feature request correctly, Carla is saying that she'd like a report (and email) specifically for people with Staff badges who accepted their Staff badge but then didn't sign up for hours even though they're supposed to do so.  Is that correct?

**Comment by cvoorhees** [comment]
@EliAndrewC  Correct. I don't know how valid of an issue it is with the other two emails that already exist, but it was something that came up in a conversation. 

Conceivably there are staffers who don't take hotel room space (like me) who could get a staff badge, food, merch, etc without actually working any hours this year, and could possibly be auto-imported again next year. While high-level staffers aren't who I'm worried about, we are getting to the point where there are so many staffers that that kind of thing might not get reported.

But I don't know what post-con analysis is done re: merch for example? 

Anyway just a thought.

**Comment by EliAndrewC** [comment]
Tagging with "next year" based on feedback at our latest Stops meeting.
